private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -81,12 +85,43 @@ class Tags extends React.Component<Props, State>  {
     e.preventDefault();
     if (this.addTagInput && this.addTagInput.current && this.props.template && this.props.template.tags) {
       const tag = this.addTagInput.current.value;
-      this.props.updateTags([...this.props.template.tags, tag]);
+      if(!this.props.template.tags.includes(tag)){
```

Comment:
Rather than doing 
`if (! ... ) {
}
else {
}`

Can we do 
`if (...) {
} else {}`